src/include/btmem.h

```diff
@@ -1087,6 +1087,17 @@ struct __wt_update {
                                                  /* AUTOMATIC FLAG VALUE GENERATION STOP */
     uint8_t flags;
 
+#ifdef HAVE_DIAGNOSTIC
```

Comment:
I think we can keep these always in the WT_UPDATE structure, and have it so that only `prev_durable_ts` is conditional on diagnostic builds. The memory size for the update structure should be unchanged, since we have 1 byte available in the rounding-to-8 calculation.
That will mean that we can't check against on-disk previous records in non-diagnostic builds.